README.md

```diff
@@ -244,6 +244,29 @@ $googleAdsClient = (new GoogleAdsClientBuilder())
     ->build();
 ```
 
+## Proxy configuration
```

Comment:
I think it would be great to add a parent level chapter called 'Proxy' that would include this 'Proxy configuration' one, same structure as for 'Logging'. This to add a paragraph describing the limitation coming from gRPC (environment variable) and may be add a link for reference.

Reply:
Not really sure about this. Do we need to expose an internal detail of the gRPC implementation to users? Would it really be relevant for them to know how gRPC sets its proxy configuration internally?

Reply:
You are right, no need to expose internal detail of gRPC here.